scripts/ci-e2e.sh

```diff
@@ -56,4 +56,5 @@ trap 'remove_ssh_key ${SSH_KEY_FINGERPRINT}' EXIT
 export DO_SSH_KEY_FINGERPRINT=${SSH_KEY_FINGERPRINT}
 
 make test-e2e
+
```

Comment:
Extra whitespace?

Reply:
oops :)

Reply:
I will wait for the conformance test to finish to fix this, ok?